components/datatypes/refunditem.schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/refunditem",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Refund Item",
+  "type": "object",
+  "description": "A refund associated with an order that defines the type of refund, the amount and the associated currency.",
+  "definitions": {
+    "refunditem": {
+      "properties": {
+        "xdm:transactionID": {
+          "title": "Transaction ID",
+          "type": "string",
+          "description": "The unique transaction identifier for this refund item."
+        },
+        "xdm:refundAmount": {
+          "title": "Refund Amount",
+          "type": "number",
+          "description": "The value of the refund."
+        },
+        "xdm:refundReason": {
+          "title": "Refund Reason",
+          "type": "string",
+          "description": "Reason why a refund was issued. Enumerated.",
+          "default": "other",
+          "meta:enum": {
+            "return": "Return",
+            "price_adjustment": "Price Adjustment"
+          }
+        },
+        "xdm:refundPaymentType": {
+          "title": "Refund Payment Type",
+          "type": "string",
+          "description": "The method of payment for this order. Enumerated, custom values allowed.",
+          "default": "other",
+          "meta:enum": {
```

Comment:
If you have such a long list of meta:enums here , do think it will be a good idea to also add enums to restrict the data to this list of values. Late on we can always add more if required. starting with an open ended list would later make it impossible to restrict as it's a breaking change.
Having a well defined list would also help in segmentation queries.

Reply:
Same is true for all other meta:enums

Reply:
@mjasrotia i'm probably misunderstanding this comment but doesn't the enum ensure these are the only accept values? ware you saying add `enums` for other fields? if so, which fields?

Reply:
@aniham , meta:enums is just the suggested display names for the enums. Actual enforcement is done through "enum" like below:

"enum": [
                        "not_provided",
                        "pending",
                        "in",
                        "out"
                    ],
                    "meta:enum": {
                        "not_provided": "Not provided",
                        "pending": "Pending verification",
                        "in": "Opt-in: the user explicitly consents to receiving messages.",
                        "out": "Opt-out: the user declines to receive any messages on this channel."
                    }

Reply:
aaah gotcha!! thank you @mjasrotia and fixed!!

Reply:
Could you please fix the lint issue ?

Reply:
sorry! ✅

Reply:
@aniham , I updated the branch from the master and I see some conflicts. Could you please resolve the conflicts on your PR ?